backend/inventory-service/db/init.go

```
@@ -0,0 +1,179 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"io/ioutil"
```

golangci-lint (1.64.8)

SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package [io] or package [os], and those implementations should be preferred in new code. See the specific function documentation for details. 

(staticcheck)